docs/getting-started.md

```diff
@@ -800,7 +791,7 @@ To help you achieve this, Meltano supports scheduled pipelines that can be orche
 
 _To learn more about orchestration, refer to the [Orchestration guide](/guide/orchestration)._
 
-1. Schedule a new [`meltano elt`](/reference/command-line-interface#elt) pipeline to be invoked on an interval using [`meltano schedule`](/reference/command-line-interface#schedule):
+1. Schedule a new pipeline to be invoked on an interval using [`meltano schedule`](/reference/command-line-interface#schedule):
```

Comment:
In https://github.com/meltano/meltano/pull/6121/files @cjohnhanson is reworking things to bias towards a scheduled job. Should we do that here too, or do you think that bloats the Getting Started guide?

Reply:
@pandemicsyn yeah I thought about it and I figured we'd want to do that at some point but I was leaving it for another PR. I was trying to keep this schedule section the same but remove ELT references so more new users only use run/invoke.